### PR TITLE
Freshen up

### PR DESCRIPTION
**Pull request by JustinAiken**
- [x] Fill in README with usage instructions 
- [x] Set up CI
  - [x] TravisCI for test suite
  - [x] Coveralls
  - [x] Code Climate
- [X] Relax Rake restriction
- [x] ⬆️ Version bump!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,35 +1,105 @@
 # Biscuit
 
-Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/biscuit`. To experiment with that code, run `bin/console` for an interactive prompt.
+[![Travis](https://img.shields.io/travis/usertesting/biscuit?style=for-the-badge)](https://travis-ci.org/usertesting/biscuit) [![Coveralls github](https://img.shields.io/coveralls/github/usertesting/biscuit?style=for-the-badge)](https://coveralls.io/github/usertesting/biscuit) [![Code Climate maintainability](https://img.shields.io/codeclimate/maintainability/usertesting/biscuit?style=for-the-badge)](https://codeclimate.com/github/usertesting/biscuit)
 
-TODO: Delete this and the text above, and describe your gem
+
+This gem is a Ruby wrapper around `@dcoker`'s [biscuit library](https://github.com/dcoker/biscuit), a multi-region HA key-value store for your AWS infrastructure secrets.
```
Comment (cdimartino) [inline README.md:6]: `s/dcoker/docker/`

File: README.md [hunk 2]
```diff
@@ -1,35 +1,105 @@
...
 
 ## Development
 
 After checking out the repo, run `bin/setup` to install dependencies. Then, run `bin/console` for an interactive prompt that will allow you to experiment.
 
 To install this gem onto your local machine, run `bundle exec rake install`. To release a new version, update the version number in `version.rb`, and then run `bundle exec rake release` to create a git tag for the version, push git commits and tags, and push the `.gem` file to [rubygems.org](https://rubygems.org).
 
+## License
+
+MIT.
```
Comment (cdimartino) [inline README.md]: Should this be a [link to the MIT license](https://opensource.org/licenses/MIT)?

File: README.md [hunk 3]
```diff
@@ -1,35 +1,105 @@
...
+
+Library created by [UserTesting](https://usertesting.com)
```
Comment (cdimartino) [inline README.md:99]: This feels like an awesome place to link to the [UserTesting Developer Pages](https://developers.usertesting.com) but we'll need to create them 🤷‍♂

**Review by cdimartino — APPROVED** [review]
lgtm

File: README.md (hunk 1 above)
Comment (JustinAiken) [reply README.md:6]: Nope, it's not by docker, it's by a Mr. Coker

File: README.md (hunk 3 above)
Comment (JustinAiken) [reply README.md:99]: developers

File: README.md (hunk 1 above)
Comment (cdimartino) [reply README.md:6]: 🤦‍♂ my bad :shipit:

**Review by perryism — APPROVED** [review]
Looks good!

**Review by nevinera — APPROVED** [review]
nice and simple

File: biscuit.gemspec [hunk 4]
```diff
@@ -30,5 +22,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "bundler", "~> 1.9"
   spec.add_development_dependency "rake", "~> 10.0"
 
-  spec.add_runtime_dependency "rake", "~> 10.0"
+  spec.add_runtime_dependency "rake"
```
Comment (dakotahp) [inline biscuit.gemspec:25]: 👏🏼 I was just researching upgrading our rake gem in the orders repo and this was a major blocker. Big fan of this change.

**Review by dakotahp — APPROVED** [review]
Nice improvements

**Pull request merged**